### PR TITLE
done

### PR DESCRIPTION
**Pull request by Barash456**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+func isValidNumber(num string, base int) bool {
+	if !isNumber(num) {
+		return false
+	}
+	switch base > 0 {
+	case base <= 10:
+		for i := range num {
+			if int(num[i]) >= base+0 {
```
Comment (zayac) [inline main.go:25]: `base+0`? What's the point? Why not just `base`?

File: main.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+				return false
+			}
+		}
+
+	case base > 10:
+		for i := range num {
+			if num[i] >= byte(base+65) {
```
Comment (zayac) [inline main.go:32]: Can you use a rune instead of a random numeric value, e.g. `'0'`?

File: main_test.go [hunk 3]
```diff
@@ -0,0 +1,64 @@
+package main
+
+import "testing"
+
+func TestIsValidNumber(t *testing.T) {
+	tests := []struct {
```
Comment (zayac) [inline main_test.go:6]: Can you add a test with an empty string and base >= 2? You program should output false.

**Review by zayac — COMMENTED** [review]
Where is `go.mod` file?
